docs/examples.rst

```diff
@@ -119,21 +125,8 @@ field must be added to store the directory of the file as follows:
     }
     ?>
 
-.. code:: php
-
-    <?php
-    /*
-       In the present example, these changes would be made in:
-       src/Template/Users/add.ctp
-       src/Template/Users/edit.ctp
-    */
-    ?>
-
-    <?php echo $this->Form->create($user, ['type' => 'file']); ?>
-        <?php echo $this->Form->input('username'); ?>
-        <?php echo $this->Form->input('photo', ['type' => 'file']); ?>
-        <?php echo $this->Form->input('photo_dir', ['type' => 'hidden']); ?>
-    <?php echo $this->Form->end(); ?>
```

Comment:
I want to keep examples there because users can be easily confused as to what exactly they need to write, and then think its a bug in the plugin when they just need an example vs instructions.

Feel free to remove that one field though.